### PR TITLE
Add Docker-MCP

### PR DESCRIPTION
**Pull request by QuantGeekDev**

### CODE REVIEW

**Comment by wong2** [comment]
Please add to the last one

**Comment by QuantGeekDev** [comment]
@wong2  fixed :)

File: README.md [hunk 1]
```diff
@@ -20,7 +20,6 @@
 - **[Fetch](https://github.com/modelcontextprotocol/servers/tree/main/src/fetch)** - Web content fetching and conversion for efficient LLM usage
 
 ## Community Servers
-
```
Comment (wong2) [inline README.md:23]: Please keep this line

**Review by wong2 — CHANGES_REQUESTED** [review]

**Comment by QuantGeekDev** [comment]
@wong2 ok now definetely fixed, sorry lol

**Pull request merged**